language-support/ts/daml-ledger/BUILD.bazel

```diff
@@ -44,6 +45,16 @@ eslint_test(
     ),
 )
 
+ts_commonjs_library(
+    name = "daml-ledger.commonjs",
```

Comment:
Does this name leak into the `package.json` files we publish?

Reply:
That name didn't leak. However, I had to restructure things and now it's gone. Will expand in a dedicated comment.